src/main/java/org/broadinstitute/hellbender/tools/walkers/vqsr/Tranche.java

```diff
@@ -159,54 +158,24 @@ private static int getOptionalInteger(Map<String,String> bindings, String key, i
         }
     }
 
-    @VisibleForTesting
-    static final class TruthSensitivityMetric {
```

Comment:
Why did you choose to eliminate this abstraction? Do you think that passing around a raw array of doubles is better than passing around a Metric object?